### PR TITLE
DOC-261: Folders, Cloning, Debugger content 

### PR DESCRIPTION
**Pull request by pwseg**
### Proposed changes

Added new content on Folders, Cloning, and the Personas Debugger.

### Merge timing

ASAP

### CODE REVIEW

**Review by markzegarelli — COMMENTED** [review]
@pwseg Since these are net new, you'll need to add them to the main side nav yaml file (src/_data/sidenav/main.yml).

@stayseesong Thoughts on naming here? I know you did work previously with regards to titles and how they fit into the nav.

**Comment by stayseesong** [comment]
@pwseg for the side-nav titles, I'd use: "Organizing Audiences" and "Personas Space Debugger"

**Review by markzegarelli — APPROVED** [review]

File: src/personas/audiences/organization.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+> Disbanding a Folder won't delete any Audiences.
+
+## Moving Audiences into Folders
+
+To move an Audience to a Folder you’ve already created, follow the steps below:
+
+1. Navigate to the Audiences tab within your Personas space.
+2. Hover over the Audience you want to move.
+3. Check the selection box that appears next to the Audience name.
+4. **Optional**: repeat Steps 2 and 3 to move multiple Audiences.
```
Comment (stayseesong) [inline src/personas/audiences/organization.md:37]:
```suggestion
4. *(Optional)* Repeat Steps 2 and 3 to move multiple Audiences.
```

File: src/personas/audiences/organization.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+### Cloning an Audience inside a Space
+
+To clone an Audience within the same Space, follow the steps below:
+
+1. Navigate to the Audiences tab within your Personas space.
+2. Click the **More Options** icon next to the Audience you want to clone.
+3. From the dropdown menu, click **Clone**.
+4. Select **Current Space**, then click **Continue**.
+5. Configure the Audience, click **Preview Results**, then click **Select Destination**.
+6. **Optional**: on the next screen, connect the Audience to a Destination.  Click **Review & Create**.
```
Comment (stayseesong) [inline src/personas/audiences/organization.md:56]:
```suggestion
6. *(Optional)* On the next screen, connect the Audience to a Destination.  Click **Review & Create**.
```

File: src/personas/audiences/organization.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## Cloning Audiences
+
+Audience cloning creates a copy of your Audience.  Within Personas, you can clone an Audience within the same space, or clone an Audience to a different space.
+
+### Cloning an Audience inside a Space
+
+To clone an Audience within the same Space, follow the steps below:
+
+1. Navigate to the Audiences tab within your Personas space.
```
Comment (stayseesong) [inline src/personas/audiences/organization.md:51]:
```suggestion
1. Navigate to the **Audiences** tab within your Personas space.
```

File: src/personas/audiences/organization.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+To disband a Folder you’ve made in Personas, click the **More Options** icon and select **Disband**.   Audiences from the disbanded Folder return to your main Audience list.
+
+> info "Note"
+> Disbanding a Folder won't delete any Audiences.
+
+## Moving Audiences into Folders
+
+To move an Audience to a Folder you’ve already created, follow the steps below:
+
+1. Navigate to the Audiences tab within your Personas space.
```
Comment (stayseesong) [inline src/personas/audiences/organization.md:34]:
```suggestion
1. Navigate to the **Audiences** tab within your Personas space.
```

File: src/personas/audiences/organization.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+* Copying an Audience between testing and production spaces
+* Copying an Audience between business units
+* Copying an Audience between teams
+
+> info "Note"
+> When you clone an Audience to a different space, first verify that the target Space includes the same events and traits for the cloned Audience.
+
+To clone an Audience between Spaces, follow the steps below:
+
+1. Navigate to the Audiences tab within your Personas space.
```
Comment (stayseesong) [inline src/personas/audiences/organization.md:73]:
```suggestion
1. Navigate to the **Audiences** tab within your Personas space.
```

File: src/personas/audiences/organization.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+2. Click the **More Options** icon next to the Audience you want to clone.
+3. From the dropdown menu, click **Clone**.
+4. Select **Different Space**, choose your target Space, then click **Continue**.
+5. Configure the Audience, click **Preview Results**, then click **Select Destination**.
+6. **Optional**: on the next screen, connect the Audience to a Destination.  Click **Review & Create**.
```
Comment (stayseesong) [inline src/personas/audiences/organization.md:78]:
```suggestion
6. *(Optional)* On the next screen, connect the Audience to a Destination.  Click **Review & Create**.
```

File: src/personas/debugger.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+Next, click on the Source you want to inspect in the Debugger.
+
+![](images/personas_debugger1.png)
+
+The Debugger displays the payload in two tabs:
+
+* **Pretty view** shows the actual API call Segment sends to your Destination.
+* **Raw view** shows the full JSON object Segment sends to your Destination from the calls you sent, including timestamps, properties, traits, and ids.
+
+Similar to the Connections Debugger, the Personas Debugger allows you to search through events, using information contained within the event's payload.
```
Comment (stayseesong) [inline src/personas/debugger.md:26]:
```suggestion
Similar to the Connections Debugger, the Personas Debugger allows you to search through events using information contained within the event's payload.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/personas/audiences/organization.md (hunk 6 above)
Comment (pwseg) [reply src/personas/audiences/organization.md:78]: Only just seeing these now - great catch(es).  Committing them to our weekly fixes branch.

**Pull request merged**